src/controllers/userController.js

```diff
@@ -0,0 +1,26 @@
+const { catchAsync } = require("../utils/errors");
+const userService = require("../services/userService");
+
+const signin = catchAsync(async (req, res) => {
+  const { KAKAO_CODE } = req.body;
+  const { jwtToken, flag, info } = await userService.signinWithKakao(KAKAO_CODE);
+  return res.status(200).json({ jwtToken, flag, info });
+});
+const signup = async (req, res) => {
```

Comment:
```suggestion
});

const signup = async (req, res) => {
```
함수간 간격을 띄워주면 가독성에 도움이 될 것 같습니다~